### ISSUE TITLE
Question: Support of broadcast/beacon

### ISSUE THREAD

**Issue by Keelback** [randdusing/cordova-plugin-bluetoothle#291]
Hello.
is it possible to act as a broadcaster (beacon). does .startAdvertising do this?

greetings
maik

**Comment by randdusing** [comment]
Yes startAdvertising will allow that. iOS may block it since they have separate classes dedicated to iBeacons, which aren't implemented in this plugin.

**Comment by Keelback** [comment]
Ok. Are you planning to add iOS support?

**Comment by randdusing** [comment]
Not if it requires the CoreLocation classes since there's already an iBeacon plugin

**Comment by Keelback** [comment]
Pity. CodenameOne adapted your PlugIn... and so i am limited to this.

**Comment by randdusing** [comment]
I wouldn't let that stop you from trying
